<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>1258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>cx</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Ax</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1110</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Restrictions</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>d(k,j)=min{(x(k,j))</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>,</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>},j=1,...,n; p&gt;=1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>(</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>x(0)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>x(n)</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>count</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>simplex method results</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>x(n)</string>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>count</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>affing-scaling method results</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>signs</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>b</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>max(min)</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>15</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>56</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>max</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>min</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_12">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>free c</string>
    </property>
    <widget class="QTextEdit" name="textEdit_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>10</y>
       <width>20</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Condition: x &gt;= 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
